microcosm_daemon/standby.py

```diff
@@ -0,0 +1,77 @@
+"""
+Standby state machine.
+
+"""
+from abc import ABCMeta, abstractproperty
+from six import add_metaclass
+
+from microcosm_daemon.sleep_policy import SleepNow
+
+
+class StandByGuard(object):
```

Comment:
We need a way for existing states to check whether they should transition to stand by. This guard wraps another state; when the state exits, it checks the stand by condition and then (maybe) transitions.